src/CBT.Core.UnitTests/Types.cs

```diff
@@ -0,0 +1,89 @@
+﻿
+using System;
+using System.Linq;
+
+// TODO: publish nuget package for MSBuildProjectBuilder to get types from it and so we can use the project builder for test project construction.
```

Comment:
reminder to self to file a defect and drop the comment if this is still relevant when this change comes in.